### PR TITLE
error_message: fix misleading error message for `pt` online-ddl-scheme

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What problem does this PR solve? 
#702

### What is changed and how it works?
change error instance name and error message

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

 - Has exported variable/fields change

### CODE REVIEW

File: pkg/terror/error_list.go [hunk 1]
```diff
@@ -867,9 +867,9 @@ var (
 	ErrSyncerUnitGhostOnlineDDLOnGhostTbl   = New(codeSyncerUnitGhostOnlineDDLOnGhostTbl, ClassSyncUnit, ScopeInternal, LevelHigh, "online ddls on ghost table `%s`.`%s`")
 	ErrSyncerUnitPTApplyEmptyTable          = New(codeSyncerUnitPTApplyEmptyTable, ClassSyncUnit, ScopeInternal, LevelHigh, "empty tables not valid")
 	ErrSyncerUnitPTRenameTableNotValid      = New(codeSyncerUnitPTRenameTableNotValid, ClassSyncUnit, ScopeInternal, LevelHigh, "tables should contain old and new table name")
-	ErrSyncerUnitPTRenameToGhostTable       = New(codeSyncerUnitPTRenameToGhostTable, ClassSyncUnit, ScopeInternal, LevelHigh, "rename table to ghost table %s not supported")
-	ErrSyncerUnitPTRenameGhostTblToOther    = New(codeSyncerUnitPTRenameGhostTblToOther, ClassSyncUnit, ScopeInternal, LevelHigh, "rename ghost table to other ghost table %s not supported")
-	ErrSyncerUnitPTOnlineDDLOnGhostTbl      = New(codeSyncerUnitPTOnlineDDLOnGhostTbl, ClassSyncUnit, ScopeInternal, LevelHigh, "online ddls on ghost table `%s`.`%s`")
+	ErrSyncerUnitPTRenameToPTTable          = New(codeSyncerUnitPTRenameToPTTable, ClassSyncUnit, ScopeInternal, LevelHigh, "rename table to pt table %s not supported")
```
Comment (csuzhangxc) [inline pkg/terror/error_list.go:870]: the previous word _ghost_ has a meaning of _temporary_ (or _not real_).

just replace it with _pt_ may not accurate?

Comment (GMHDBJD) [reply pkg/terror/error_list.go:870]: Did you mean we should replace both ghost and pt to another word such as temporary or shadow?

Comment (csuzhangxc) [reply pkg/terror/error_list.go:870]: or maybe _ghost temporary_ / _pt temporary_?

File: pkg/terror/error_list.go [hunk 2]
```diff
@@ -862,14 +862,14 @@ var (
...
-	ErrSyncerUnitPTRenameToGhostTable       = New(codeSyncerUnitPTRenameToGhostTable, ClassSyncUnit, ScopeInternal, LevelHigh, "rename table to ghost table %s not supported")
-	ErrSyncerUnitPTRenameGhostTblToOther    = New(codeSyncerUnitPTRenameGhostTblToOther, ClassSyncUnit, ScopeInternal, LevelHigh, "rename ghost table to other ghost table %s not supported")
-	ErrSyncerUnitPTOnlineDDLOnGhostTbl      = New(codeSyncerUnitPTOnlineDDLOnGhostTbl, ClassSyncUnit, ScopeInternal, LevelHigh, "online ddls on ghost table `%s`.`%s`")
+	ErrSyncerUnitPTRenameToPTTable          = New(codeSyncerUnitPTRenameToPTTable, ClassSyncUnit, ScopeInternal, LevelHigh, "rename table to pt temporary table %s not supported")
+	ErrSyncerUnitPTRenamePTTblToOther       = New(codeSyncerUnitPTRenamePTTblToOther, ClassSyncUnit, ScopeInternal, LevelHigh, "rename pt temporary table to other temporary table %s not supported")
+	ErrSyncerUnitPTOnlineDDLOnPTTbl         = New(codeSyncerUnitPTOnlineDDLOnPTTbl, ClassSyncUnit, ScopeInternal, LevelHigh, "online ddls on pt temporary table `%s`.`%s`")
```
Comment (csuzhangxc) [inline pkg/terror/error_list.go:872]: Can we update this message to something like

```
online ddl metadata for table `%s`.`%s` not found
```

if so, update for gh-ost too.

File: _utils/terror_gen/errors_release.txt [hunk 3]
```diff
@@ -273,14 +273,14 @@ ErrSyncerUnitOnlineDDLSchemeNotSupport,[code=36040:class=sync-unit:scope=interna
...
 ErrSyncerUnitPTApplyEmptyTable,[code=36047:class=sync-unit:scope=internal:level=high],"empty tables not valid"
 ErrSyncerUnitPTRenameTableNotValid,[code=36048:class=sync-unit:scope=internal:level=high],"tables should contain old and new table name"
-ErrSyncerUnitPTRenameToGhostTable,[code=36049:class=sync-unit:scope=internal:level=high],"rename table to ghost table %s not supported"
-ErrSyncerUnitPTRenameGhostTblToOther,[code=36050:class=sync-unit:scope=internal:level=high],"rename ghost table to other ghost table %s not supported"
-ErrSyncerUnitPTOnlineDDLOnGhostTbl,[code=36051:class=sync-unit:scope=internal:level=high],"online ddls on ghost table `%s`.`%s`"
+ErrSyncerUnitPTRenameToPTTable,[code=36049:class=sync-unit:scope=internal:level=high],"rename table to pt temporary table %s not supported"
+ErrSyncerUnitPTRenamePTTblToOther,[code=36050:class=sync-unit:scope=internal:level=high],"rename pt temporary table to other temporary table %s not supported"
+ErrSyncerUnitPTOnlineDDLOnPTTbl,[code=36051:class=sync-unit:scope=internal:level=high],"online ddl metadata for table `%s`.`%s` not found"
```
Comment (WangXiangUSTC) [inline _utils/terror_gen/errors_release.txt:283]: the error log  same with `ErrSyncerUnitGhostOnlineDDLOnGhostTbl`, maybe we can only use one or add `gh-ost` and `pt` in error log

Comment (GMHDBJD) [reply _utils/terror_gen/errors_release.txt]: address in e1ba8ba

**Comment by csuzhangxc** [comment]
@GMHDBJD please cherry-pick it manually.

**Pull request merged**